[2024-11-14T07:09:21.274+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-14T07:09:21.502+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: model_training_update.update_data manual__2024-11-14T07:09:14.542180+00:00 [queued]>
[2024-11-14T07:09:21.608+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: model_training_update.update_data manual__2024-11-14T07:09:14.542180+00:00 [queued]>
[2024-11-14T07:09:21.613+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-14T07:09:21.809+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): update_data> on 2024-11-14 07:09:14.542180+00:00
[2024-11-14T07:09:21.885+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=288) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-14T07:09:21.944+0000] {standard_task_runner.py:72} INFO - Started process 295 to run task
[2024-11-14T07:09:21.917+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'model_training_update', 'update_data', 'manual__2024-11-14T07:09:14.542180+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmp3m50dqec']
[2024-11-14T07:09:21.996+0000] {standard_task_runner.py:105} INFO - Job 5: Subtask update_data
[2024-11-14T07:09:22.122+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-11-14T07:09:23.048+0000] {task_command.py:467} INFO - Running <TaskInstance: model_training_update.update_data manual__2024-11-14T07:09:14.542180+00:00 [running]> on host 61d1a0d6a190
[2024-11-14T07:09:25.208+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='model_training_update' AIRFLOW_CTX_TASK_ID='update_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-14T07:09:14.542180+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-14T07:09:14.542180+00:00'
[2024-11-14T07:09:25.231+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-14T07:09:29.702+0000] {python.py:240} INFO - Done. Returned value was: {'message': 'Données mises à jour avec succès.'}
[2024-11-14T07:09:30.060+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-14T07:09:30.083+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=model_training_update, task_id=update_data, run_id=manual__2024-11-14T07:09:14.542180+00:00, execution_date=20241114T070914, start_date=20241114T070921, end_date=20241114T070930
[2024-11-14T07:09:30.427+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-14T07:09:30.690+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-14T07:09:30.720+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
